### ISSUE TITLE
Y24-297 - Change 'Offset tags by' wording in Limber tag plate selection

### ISSUE THREAD

**Issue by neilsycamore** [sanger/limber#1889]
As Team Lead 10X & Custom Sequencing (Jo) we would like, when selecting tag groups, to have the 'Offset tags by' changed to 'Start Tags from' and the tag number selected should be the tag present in the first well. This should not be specific to a plate but to all plates when using the tag plate stage in Limber.

This is to avoid confusion and delay whilst a service ticket is raised for the tags to be changed

Example from the lab team:
![10X Offset tags](https://github.com/user-attachments/assets/b10b9811-425a-43ce-ba6a-5472f33ebba6)

Just before we create a Ligation XP plate for 10x semi-auto, we create a custom tag plate where we assign tags to the plate wells. Sometimes when we have to choose tags starting from any number other than '1', we select the option of ,offset tags, and write a number there. As you canb see in the screenshot, I needed the tags to start from '10', so in the offset tags column I wrote '9'. A lot of times, this leads to tag shift issues. So, is it possible to change it on limber, to be more clearer?

Currently it says: "Offset tags by: 9" = Tags start from 10
Can we change it to: "Start Tags from: 10" = Tags start from 10

### Acceptance Criteria:

- [ ]

**Comment by JoCartwright** [comment]
Terrific, Thanks!

**Comment by SujitDey2022** [comment]
![Image](https://github.com/user-attachments/assets/0586debf-9831-42fb-9921-78c06455a623)

**Comment by TWJW-SANGER** [comment]
@SujitDey2022  It might be worth identifying the reason/s the team need to "when we have to choose tags starting from any number other than '1'".

**Comment by SujitDey2022** [comment]
> @SujitDey2022 It might be worth identifying the reason/s the team need to "when we have to choose tags starting from any number other than '1'".

Yes, I had a meeting with the SR Team yesterday and will be doing the same.

**Comment by sambucaboywicked** [comment]
These are the tag sets we use:
10X_Plate TT Set A i5
10X_Plate TT Set A i7

I can't seem to find the page on Sequencescape anymore with the tags listed?

**Comment by TWJW-SANGER** [comment]
Placing on hold.

It is worth noting that desired information that "Tags start from 10" is already displayed on the plate layout image on the screenshot already...... 

The 2 goals here are:
* To avoid tag shift errors. 
* To use up the volume of the tag plates.

What possible options can we think of for this?  

- We could do as the story suggests change from offset to absolute tag position and make this size of this story bigger to account for additional error checking required due to tag sets starting from different indices.
- We could simply (?) add some bold text underneath the offset edit box to state "Tags will start from 10"  - given the page must know this somewhere already to display the layout. 
- Something else... Are there other places where a tag offset error could be detected or the user directed to check & confirm in other parts of the application?

@sambucaboywicked Would you be happy with the second solution (additional text) or is this worth discussing further?

**Comment by sambucaboywicked** [comment]
Hey Thomas,

I've spoken to our team, and we are happy with option 2: _We could simply (?) add some bold text underneath the offset edit box to state "Tags will start from 10" - given the page must know this somewhere already to display the layout._
Is this pausible?

King Regards,
Meg

**Comment by SujitDey2022** [comment]
@TWJW-SANGER is this item pending any further clarification? Can we remove the on-hold tag from it?

Regards,

**Comment by neilsycamore** [comment]
@sambucaboywicked 
Hi Meg,
This story has been moved into UAT and is available for testing.
I have used an existing UAT plate (SQPU-193892-W) to step through to the tagging screen and to me the changes requested are present. 
Could someone test that this now fits the criteria: 'add some bold text underneath the offset edit box to state "Tags will start from 10" '?
Whoever tests will need to have logged into SS UAT first (https://uat.sequencescape.psd.sanger.ac.uk/) to be able to use Limber https://uat.limber.psd.sanger.ac.uk/

**Comment by me12-WSI** [comment]
Hi Neil, 
Has this also been changed on the manual pipeline or only Auto as we have not seen this pop up when doing manual runs?
Kind Regards,
Meg
